enhanced_rag/azure_integration/automation/reindex_manager.py

```
@@ -1,23 +1,24 @@
-previously duplicated implementations. It uses REST operations primarily
-and falls back to legacy ReindexOperations only when needed.
-
-DEPRECATION NOTICE: This module consolidates functionality that was previously
-duplicated between ReindexOperations and ReindexAutomation.
+This module provides the unified reindexing interface using REST operations
+only. It replaces legacy SDK-based flows and removes the dependency on
+ReindexOperations.
 """
 
 import logging
 import warnings
```

Ruff (0.12.2)

`warnings` imported but unused

Remove unused import: `warnings`

(F401)